documentation/book/cluster-operator.adoc

```diff
@@ -722,6 +765,26 @@ The resources created by the Cluster Operator into the {ProductPlatformName} clu
 [[kafka_connect_configuration_json_config]]
 ===== Kafka Connect configuration
 
+The `logging` field allows configuration of loggers. These loggers are:
+`
+log4j.rootLogger
+connect.root.logger.level
+log4j.logger.org.apache.zookeeper
+log4j.logger.org.I0Itec.zkclient
+log4j.logger.org.reflections
+`
+The setting can be done in two ways. User can change arbitrary log level by specifying key value JSON. Second way is to use custom config map.
+It can be achieved by this setting:
+`{
+    "fromConfigMap" :
+        {
+            "name" : "customConfigMap"
+        }
+ }
+`
+The difference between these two options is that second one is not validated. That means user can change in logging setting whatever he wants.
+First option supports default values.
+
```

Comment:
The difference between these two options is that latter is not validated and does not support default values. That means that a user can supply any logging configuration, even if it is incorrect.